test/runmoitests.jl

```diff
@@ -31,33 +31,34 @@ end
     println("\nstarting MOI.Test tests")
     # TODO test other real types
     T = Float64
-    model = MOI.Utilities.CachingOptimizer(
-        MOI.Utilities.UniversalFallback(MOI.Utilities.Model{T}()),
-        Hypatia.Optimizer{T}(; default_tol_relax = 4),
+    model = MOI.Bridges.full_bridge_optimizer(
+        MOI.Utilities.CachingOptimizer(
+            MOI.Utilities.UniversalFallback(MOI.Utilities.Model{T}()),
+            Hypatia.Optimizer{T}(; default_tol_relax = 4),
+        ),
+        T,
     )
     MOI.set(model, MOI.Silent(), true)
-
-    tol = 2 * sqrt(sqrt(eps(T)))
-    config = MOI.Test.Config(
-        T,
-        atol = tol,
-        rtol = tol,
-        exclude = Any[
-            MOI.ConstraintBasisStatus,
-            MOI.VariableBasisStatus,
-            MOI.ObjectiveBound,
+    MOI.Test.runtests(
+        model,
+        MOI.Test.Config(
+            T,
+            atol = 2 * sqrt(sqrt(eps(T))),
+            rtol = 2 * sqrt(sqrt(eps(T))),
+            exclude = Any[
+                MOI.ConstraintBasisStatus,
+                MOI.VariableBasisStatus,
+                MOI.ObjectiveBound,
+                MOI.SolverVersion,
+            ],
+        ),
+        exclude = String[
+            # TODO(odow): unexpected failure. But this is probably in the bridge
+            # layer, not Hypatia.
+            "test_model_UpperBoundAlreadySet",
+            "test_model_LowerBoundAlreadySet",
```

Comment:
@chriscoey the `copy_to` test was failing because it used a constraint type not supported by Hypatia. I added a bridge.

But this caused these tests to fail, which is probably because of https://github.com/jump-dev/MathOptInterface.jl/issues/1665.

Reply:
Ok 👍